src/store/actions.ts

```diff
@@ -1,6 +1,8 @@
-export * from './auth';
-export * from './skill';
+export { actions as authActions } from './auth';
+export { actions as skillActions } from './skill';
+export { actions as educationActions } from './education';
+export { actions as experienceActions } from './experience';
 export * from './notification';
 export * from './okr';
-export * from './experience';
+export * from './education';
 export * from './opportunities';
```

Comment:
Why do we need to change import format for `auth` and others? Also there are seems to be redundant import from `education`